src/data/nat/choose/sum.lean

```diff
@@ -103,6 +103,31 @@ begin
   simpa [sum_range_choose_halfway n] using t
 end
 
+lemma four_pow_le_two_mul_add_one_mul_central_binom (n : ℕ) :
+  4 ^ n ≤ (2 * n + 1) * choose (2 * n) n :=
+begin
+  calc 4 ^ n = (1 + 1) ^ (2 * n) : begin
+      rw one_add_one_eq_two,
+      rw pow_mul,
+      suffices h : 2 ^ 2 = 4,
+      rw h,
+      dec_trivial,
+    end
+  ...        = ∑ m in range (2 * n + 1), choose (2 * n) m : by simp [add_pow 1 1 (2 * n)]
+  ...        ≤ ∑ m in range (2 * n + 1), choose (2 * n) (2 * n / 2) : begin
+      apply sum_le_sum,
+      intros i hi,
+      apply choose_le_middle i (2 * n),
+    end
```

Comment:
Similarly, this condenses to `sum_le_sum (λ i hi, choose_le_middle i (2 * n))`

Reply:
yep, I'll make that change